h2o-genmodel-extensions/xgboost/build.gradle

```diff
@@ -5,16 +5,18 @@ description = "H2O GenModel XGBoost support"
 dependencies {
     compile project(":h2o-genmodel")
 
-    // XGBoost dependencies but not officially published
-    compile files("xgblib/xgboost4j-0.7.1.jar",
-                  "xgblib/xgboost4j-native-osx_minimal-0.7.1.jar",
-                  "xgblib/xgboost4j-native-linux_minimal-0.7.1.jar",
-                  "xgblib/xgboost4j-native-linux_ompv3-0.7.1.jar",
-                  "xgblib/xgboost4j-native-linux_gpuv4-0.7.1.jar")
-
-    // These are dependencies extracted from xgboost/jvm-packages/pom.xml
-    compile 'com.esotericsoftware.kryo:kryo:2.21'
-    compile 'commons-logging:commons-logging:1.2'
+    // XGBoost dependencies published into Maven central by H2O
+    // Versioning rules may differ for XGBoost artifacts published by H2O
+    compile('ai.h2o:xgboost4j:0.7.3') {
+        exclude group: 'org.scala-lang', module: 'scala-compiler'
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+        exclude group: 'org.scala-lang', module: 'scala-library'
+        exclude group: 'com.typesafe.akka', module: 'akka-actor_2.11'
+    }
+    compile 'ai.h2o:xgboost4j-linux-gpuv4:0.7.3'
```

Comment:
shouldn't we add the gpuv3 and ompv4 as well?

Reply:
`ompv4` is not necessary because `gpuv4` switches to the right backend at runtime and loads fine if the GPU libraries are not present.

`gpuv3` is omitted on purpose - users will GPUs will most likely not have an old version of OS. I would only add it if we see the need for it.